### PR TITLE
Support using vis_bbox together with vis_instance_segmentation  

### PR DESCRIPTION
**Pull request by yuyu2172**
figure_1

### CODE REVIEW

File: chainercv/visualizations/colormap.py [hunk 1]
```diff
@@ -1,18 +1,29 @@
-def voc_colormap(label):
+import numpy as np
+
+
+def voc_colormap(labels):
     """Color map used in PASCAL VOC
 
     Args:
-        label (int): Class id.
+        labels (~numpy.ndarray): Class ids.
```
Comment (knorth55) [inline chainercv/visualizations/colormap.py]: list of int?

File: chainercv/visualizations/colormap.py [hunk 2]
```diff
@@ -5,7 +5,7 @@ def voc_colormap(labels):
     """Color map used in PASCAL VOC
 
     Args:
-        labels (~numpy.ndarray): Class ids.
+        labels (list of ints): Class ids.
```
Comment (Hakuyume) [inline chainercv/visualizations/colormap.py]: How about `iterable of ints`?

**Comment by Hakuyume** [comment]
Can we support specifying color for each class in `vis_bbox`? For example, the box for `car` is red, the box for `dog` is blue ...
We can rename the argument `color` to `instance_color` and add `class_color (label_color)`.

**Comment by yuyu2172** [comment]
figure_1-1

File: chainercv/visualizations/vis_bbox.py [hunk 3]
```diff
@@ -1,25 +1,44 @@
...
+             instance_colors=None, alpha=1., linewidth=3., ax=None):
     """Visualize bounding boxes inside image.
 
     Example:
 
-        >>> from chainercv.datasets import VOCDetectionDataset
+        >>> from chainercv.datasets import VOCBboxDataset
         >>> from chainercv.datasets import voc_bbox_label_names
         >>> from chainercv.visualizations import vis_bbox
         >>> import matplotlib.pyplot as plot
```
Comment (Hakuyume) [inline chainercv/visualizations/vis_bbox.py]: `plot` -> `plt` (https://github.com/chainer/chainer/pull/3698)

File: chainercv/visualizations/vis_bbox.py [hunk 4]
```diff
@@ -32,6 +51,15 @@ def vis_bbox(img, bbox, label=None, score=None, label_names=None, ax=None):
              This is optional.
         label_names (iterable of strings): Name of labels ordered according
             to label ids. If this is :obj:`None`, labels will be skipped.
+        instance_colors (iterable of tuple): List of colors.
```
Comment (Hakuyume) [inline chainercv/visualizations/vis_bbox.py]: `tuple` -> `tuples`

File: chainercv/visualizations/vis_bbox.py [hunk 5]
```diff
@@ -54,12 +82,20 @@ def vis_bbox(img, bbox, label=None, score=None, label_names=None, ax=None):
     if len(bbox) == 0:
         return ax
 
+    if instance_colors is None:
+        # Red
+        instance_colors = np.ones((len(bbox), 3), dtype=np.float32)
```
Comment (Hakuyume) [inline chainercv/visualizations/vis_bbox.py]: `ones` -> `zeros` ?

File: chainercv/visualizations/vis_bbox.py (hunk 3 above)
Comment (yuyu2172) [reply chainercv/visualizations/vis_bbox.py]: This has not been fixed yet in ChainerCV.

**Pull request merged**